src/styles/components/charts/styles.less

```diff
@@ -1,342 +1,580 @@
+      flex-direction: column;
+      justify-content: center;
+      left: 0;
+      position: absolute;
+      text-align: center;
+      top: 0;
+      width: 100%;
+
+      .icon {
+        display: inline-block;
+      }
+
+    }
+
+  }
 
+  /**********************************/
+  /*  Specific to Nodes Grid View   */
+  /**********************************/
+  .nodes-grid-dials {
     display: flex;
-    justify-content: center;
-    margin-top: 10px;
+    flex-flow: row wrap;
+    justify-content: space-around;
+    margin-left: -(ceil(@base-spacing-unit * 0.3333));
+    margin-right: -(ceil(@base-spacing-unit * 0.3333));
+
+    .nodes-grid-dials-item {
+      height: ceil(@base-spacing-unit * 3.33);
+      margin-bottom: ceil(@base-spacing-unit * 0.3333);
+      margin-right: ceil(@base-spacing-unit * 0.3333);
+      width: ceil(@base-spacing-unit * 3.33);
+
+      .chart-dialchart {
+
+        // Remove stroke when dials are in small containers
+        .arc path {
+          stroke-width: 1px;
+          stroke: @panel-background-color;
+        }
 
-    & > span {
+        .arc {
 
-      margin-left: 10px;
+          &.path-color-0 {
+            fill: @purple;
+            stroke: @purple;
+          }
+
+          &.path-color-1 {
+            fill: @blue;
+            stroke: @blue;
+          }
+
+          &.path-color-2 {
+            fill: @red;
+            stroke: @red;
+          }
+
+          &.path-color-3 {
+            fill: @blue;
+            stroke: @blue;
+          }
+
+          &.path-color-4 {
+            fill: @yellow;
+            stroke: @yellow;
+          }
+
+          &.path-color-5 {
+            fill: @green;
+            stroke: @green;
+          }
+
+          &.path-color-6 {
+            fill: @pink;
+            stroke: @pink;
+          }
 
-      &.disabled {
+          &.path-color-7 {
+            fill: @cyan;
+            stroke: @cyan;
+          }
 
-        opacity: 0.7;
+          &.path-color-8 {
+            fill: color-lighten(@neutral, 40);
+            stroke: color-lighten(@neutral, 40);
+          }
+
+          &.path-color-unused {
+            fill: color-lighten(@neutral, 5);
+            stroke: color-lighten(@neutral, 5);
+          }
+
+        }
+
+        .unit {
+          color: @white;
+          font-size: 16px;
+        }
+
+        .unit-label {
+          font-size: 11px;
+        }
+
+        .error {
+          position: relative;
+          top: -2px;
+        }
 
       }
 
-      &:hover {
+    }
+
+    .nodes-grid-dials-spacer {
+      height: 0;
+      margin-right: ceil(@base-spacing-unit * 0.3333);
+      width: ceil(@base-spacing-unit * 3.33);
+    }
+
+  }
+
+}
+
+& when (@charts-enabled) and (@layout-screen-small-enabled) {
+
+  @media (min-width: @layout-screen-small-min-width) {
+
+    .chart {
+
+      .unit-bordered-horizontal-mini {
+        border-right: 1px solid @chart-axis-color;
+        margin-top: @base-spacing-unit-screen-small * 1/4;
```

Comment:
I'll bring this up to the frontend team in our forthcoming code styling discussion. I'd vote for fractions everywhere :)